src/Entities/Transaction.php

```diff
@@ -22,14 +22,11 @@ final class Transaction extends Entity
 
     public const TYPE_SALE = 'sale';
 
-    public const TYPE_CREDIT = 'credit';
```

Comment:
It's still used in responses, is it?

Reply:
I don't see any issues in returning the type. Odd thing is that I can't find any usages of `TYPE_*` or `types()` other than in `TestCase::getFakeValue()`.

Reply:
Consider the integration that uses this SDK and want to write something like
~~~php
$transaction = $client->transacitons->get($id);
if ($transaction->getType() === Transaction::TYPE_CREDIT) {
 // ...
}
~~~